content/swift/concepts/optionals/optionals.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: 'Optionals'
+Description: 'Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Optionals'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+Optionals are types that deal with scenarios involving the potential presence or absence of a value (nil).
+
+## Syntax
+
+In Swift, a non-optional variable can't be set to nil. Rather, the optional type must be defined using the added question mark `?`.
+
+Below, a variable `myInt` is declared and set to the optional `Int` type, `Int?`. It can either be equal to any integer or nil.
```

Comment:
I don't see a variable called `myInt` declared, just `maybeInt`?

Reply:
😮 you're right! It should be `maybeInt`